app/controllers/agents_controller.rb

```diff
@@ -95,7 +105,17 @@ def update
     alert_id = agent_alert_container_id(agent, parent_id)
     deletable = params[:deletable]&.eql?('true')
     if response_error?(agent_update)
-      render_turbo_stream(alert_error(id: alert_id) { JSON.pretty_generate(response_errors(agent_update)) })
+      errors = []
```

Comment:
this part of the code is duplicated above, create a function if needed.

Reply:
ok